spec/graphql/queries/users/get_users_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Types::QueryType do
+  describe 'display users' do
+    it 'can query all users' do
+      create(:user, name: "Bruce Easley")
+      create_list(:user, 4)
+
+      result = PairedBeSchema.execute(query).as_json
+      expect(result["data"]["getUsers"].count).to eq(5)
+      expect(result["data"]["getUsers"].first["name"]).to eq("Bruce Easley")
+
+      users = User.all
+      expect(result.dig("data", "getUsers")).to match_array(
+      users.map { |user| {"mod" => user.mod, "name" => user.name, "program" => user.program} })
```

Comment:
Cool way of doing this!

Reply:
Can't take credit. It was in a tutorial!